### ISSUE TITLE
"Invisible zombie" glitch

### ISSUE THREAD

**Issue by fishingsman** [SmartlyDressedGames/Unturned-3.x-Community#3100]
No, not that zombies that are like transparent purple, real invisible zombies, that have no model, and dont even make any sound, this maybe created by ping issues, but this is hella strange.

**Comment by GazziFX** [comment]
Its well known bug because when player leaves region it locally unloads and then if player reverts back this region does not load again, I think region unload should be authoritative

**Comment by QERT2002** [comment]
Yes, it's very annoying to play.Why not make the region load every 10 seconds?

**Comment by SDGNelson** [comment]
Interesting/fair point about the authoritative region change, but in this case is it on a custom map? One common issue is overlapping navmeshes - the client only loads zombies in a single navmesh at a time.

**Comment by QERT2002** [comment]
I have this glitch on the all maps, if you running into region and your ping go up, zombie dont spawn on client, but spawn on server and damage your when you cant it. I fix this by running away from the location and returning back , but this is very terrible, also sometimes players' buildings are not loaded and you either rejoin the server or run away from the location and returning back. Please fix this glitch.

**Comment by SDGNelson** [comment]
Keeping this as the oldest currently open note of the problem. No ETA unfortunately.

I think best thing to do here thinking ahead to when zombies hopefully have full pathfinding would be to ignore the "nav" stuff when replicating zombies. The server should send zombies within a certain area regardless of current navmesh.
